library/network_connections.py

```diff
@@ -1408,7 +1416,8 @@ def check_mode_set(self, check_mode, connections=None):
             or (c == CheckMode.PRE_RUN and check_mode in [CheckMode.REAL_RUN])
             or (c == CheckMode.REAL_RUN and check_mode in [CheckMode.DONE])
             or (c == CheckMode.DRY_RUN and check_mode in [CheckMode.DONE])
-        )
+        ):
+            raise AssertionError("check_mode value is incorrect {0}".format(c))
```

Comment:
should this be `raise AssertionError("updating check_mode value from {0} into {1} is incorrect".format(c, check_mode))` ?

Reply:
fixed https://github.com/linux-system-roles/network/pull/362/commits/2bfe053a5a4011a8002cd5ec43e62b7a8489ed80#diff-6b3f5f6149ac1330c5e2c832286b63b3597158a998f2244ddbec2045ace1f794R1420